src/main/java/com/iemr/mmu/controller/labtechnician/LabTechnicianCreateController.java

```diff
@@ -39,16 +39,14 @@
 import io.swagger.annotations.ApiOperation;
 
 /***
- * 
- * @author NE298657
- * @Objective Fetching lab tests prescribed by doctor
- *
+ * @Objective Saving lab test results given by LabTechnician
  */
 
 @RestController
 @CrossOrigin
 @RequestMapping(value = "/labTechnician", headers = "Authorization")
-public class LabtechnicianFetchController {
+public class LabTechnicianCreateController {
```

Comment:
@swetatech24 This class name should be LabTechnicianController. Please correct and commit